barretenberg/cpp/src/barretenberg/vm/avm_trace/AvmMini_execution.cpp

```diff
@@ -164,7 +49,7 @@ std::vector<Row> Execution::gen_trace(std::vector<Instruction> const& instructio
 {
     AvmMiniTraceBuilder trace_builder{};
```

Comment:
`{}` also not needed, the constructor is always called for classes. (in the same way, `()` is usually not needed)

Reply:
Ok, I will look for the remaining ones and remove them.
FYI, I got use to this bad habit because for the field (often denoted Fr) type, we have a constructor which is not called by default is requires the braces when you want an array/vector to be initialized.